lib/src/layer/tile_layer/tile_bounds/tile_bounds.dart

```diff
@@ -124,30 +123,30 @@ class WrappedTileBounds extends TileBounds {
 
     final tzDouble = zoom.toDouble();
 
-    Tuple2<int, int>? wrapX;
+    (int, int)? wrapX;
     if (crs.wrapLng != null) {
```

Comment:
Same here, use pattern matching:
```dart
if (crs.wrapLng case final wrapLng?) {
  final wrapXMin = (crs.latLngToPoint(LatLng(0, wrapLng.$1), tzDouble).x / _tileSize).floor();

  // ...
}
```

Reply:
Resolved in https://github.com/fleaflet/flutter_map/pull/1517/commits/3a58ee8354ff119a5168f94eef54aa1ac6a2f373